daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/PhaseOne.scala

```diff
@@ -712,9 +712,18 @@ private[lf] final class PhaseOne(
         compileExp(env, arg) { arg =>
           Return(t.CreateDefRef(tmplId)(arg))
         }
-      case UpdateCreateInterface(_, arg) =>
-        compileExp(env, arg) { arg =>
-          Return(SBResolveCreate(arg))
+      case UpdateCreateInterface(ifaceId, arg) =>
+        unaryFunction(env) { (tokPos, env) =>
```

Comment:
did you deliberately decide against moving it to a top-level function or just something you'll do separately?

Reply:
I decide to not decide anything. 
We will see later if we move it to the top level. 
In the same time we could see if we factorize (as a top level function) the `SEPreventCatch(SBViewInterface(ifaceId)(env.toSEVar(payloadPos)))`